Makefile

```diff
@@ -21,14 +21,17 @@ all: wheel/dasmon wheel/webmon wheel/workflow SNSdata.tar.gz
 
 create/conda:  ## create conda environment "webmon" with file conda_environment.yml
 	conda env create --name webmon --file conda_environment.yml
+	conda env update --name webmon --file conda_development.yml
 
 create/mamba:  ## create conda environment "webmon" with file conda_environment.yml using mamba
 	conda env create --name webmon python=$(PYTHON_VERSION)
 	conda install --name base -c conda-forge mamba
 	mamba env update --name webmon --file conda_environment.yml
+	mamba env update --name webmon --file conda_development.yml
 
 docker/pruneall: ## stop and force remove all containers, images and volumes
 	docker system prune --force --all --volumes
+	docker system prune --force --all --volumes
```

Comment:
repeated line?

Reply:
I think I didn't solve the conflicts in the `Makefile` as good as I thought :facepalm: , thanks for catching this